tmk_core/chibios.mk

```diff
@@ -198,10 +198,13 @@ ifneq ("$(SERIAL)","")
 	DFU_ARGS += -S $(SERIAL)
 endif
 
+ST_LINK_ARGS ?=
+
 # List any extra directories to look for libraries here.
 EXTRALIBDIRS = $(RULESPATH)/ld
 
 DFU_UTIL ?= dfu-util
+ST_LINK_CLI ?= st-link_cli
```

Comment:
Could you adjust `st-link_cli` to be `st-link-cli` here as well?

Reply:
This line is setting command or path.
We can't modify.

Reply:
Well, rather, it looks like it's a file name, basically, like dfu-util.

In fact, downloaded this on my system (windows), and it's explicitly named this, actually: `ST-LINK_CLI.exe`. 

So this should be fine.

Reply:
Ah, ok. Thanks!